src/view/uniprotkb/components/ProteinNamesView.tsx

```diff
@@ -32,24 +30,191 @@ export const ProteinNames: React.FC<ProteinNamesDataProps> = ({
   return <NameView {...props} />;
 };
 
-export const EntryProteinNames: React.FC<ProteinNamesDataProps> = ({
-  recommendedName,
-  shortNames,
-  alternativeNames,
+const NameWithEvidence: React.FC<{ data: ValueWihEvidence }> = ({ data }) => (
+  <Fragment>
+    {data.value}{' '}
```

Comment:
This can be written like this as well, but the white-space in the end might not be very obvious to the developer:

    {`${data.value} `}